src/utils/helper.ts

```diff
@@ -62,12 +55,6 @@ export const scoringByCategory = {
     1: "The company participates in animal testing, but is actively looking for alternatives",
```

Comment:
I can't comment my suggestion for line 52, but it should be changed to `[ScoreCategories.AnimalTesting]: {`